content/javascript/concepts/dates/terms/setUTCHours/setUTCHours.md

```diff
@@ -0,0 +1,63 @@
+---
+Title: '.setUTCHours()'
+Description: 'Sets the hour value of a date object using UTC.'
+Subjects:
+  - 'Web Development'
+  - 'Computer Science'
+Tags:
+  - 'Date'
+  - 'Methods'
+CatalogContent:
+  - 'introduction-to-javascript'
+  - 'paths/front-end-engineer-career-path'
+---
+
+The **`.setUTCHours()`** method sets the hour value of a `Date` object according to [UTC](https://en.wikipedia.org/wiki/Coordinated_Universal_Time) and returns the updated `Date` object.
+
+Optionally, you can use the method to also set the date's minute, second, and millisecond UTC values. These are the valid number ranges for the time units:
```

Comment:
Move all of this about the optional parameters to the end of the Syntax section.